<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGraphs</class>
 <widget class="QMainWindow" name="QtGraphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGraphs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_AddNode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить узел</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddEdge">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить грань</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteNode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить узел</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteEdge">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить грань</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ObxodInDepth">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Обход в глубину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AlgoritmDijkstra">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>50</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Алгортм Дейкстры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ObxodInWidht">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Обход в ширину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ZagotovlenGraph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Заготовленный граф</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_randGraph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Случайный граф</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UpdateEdge">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить грань</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CLEAR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>очистка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_addDelNodeValue">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_FirstNode">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_SecondNode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Weight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAlgoritmObxodaInDepth">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAlgoritmDijkstraFirst">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAlgoritmDijkstraSecond">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ObxodInWidht">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tablica">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Печать 
таблицы смежности</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
